lib/migrator/index.js

```diff
@@ -40,6 +40,7 @@ const addReporter = async (tag, link) => {
 }
 
 const filesToDelete = [
+  '.port.tmp',
```

Comment:
Should we move it rather than delete it?

Reply:
Really it should have been deleted when the app closed last:

https://github.com/alphagov/govuk-prototype-kit/blob/v13/lib/build/tasks.js#L152-L159

It's an ephemeral file that shouldn't hang around, I just thought it should be deleted just in case it's still there somehow, so it doesn't accidentally get committed.